MovieSharing/VideoModel.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Video.swift
+//  MovieSharing
+//
+//  Created by Mariam Korashi on 28.09.21.
+//
+
+import Foundation
+
+struct VideosResponse:Decodable {
+    
+    var items: [Video]
+    
+}
+
+struct Video:Decodable {
+    
+    var id: String
+    var snippet:Snippet
+    
+}
+
+
+struct Snippet:Decodable {
+    
+    var publishedAt: String
+    var title:String
+        var description: String
+        var Thumbnails: Thumbnail
```

Comment:
since a snippet contains just one thumbnail, the var of type thumbnail should better be named thumbnail. "thumbnails" kind of would make one expect a collection here